### PR TITLE
Add option to ignore diffs across the system

### PR DESCRIPTION
**Pull request by jgfouca**
Useful for performance testing.

Test suite: btr and unit tests
Test baseline:
Test namelist changes:
Test status: bit for bit

User interface changes?:

Update gh-pages html (Y/N)?:

### CODE REVIEW

**Comment by rljacob** [comment]
What does "ignore diffs across the system" mean?

**Comment by jgfouca** [comment]
@rljacob , the option for the testing infrastructure (create_test, Jenkins, etc) to ignore history diffs.

**Review by jedwards4b — APPROVED** [review]

File: CIME/bless_test_results.py [hunk 1]
```diff
@@ -216,7 +216,21 @@ def bless_test_results(
...
 ):
-    bless_all = not (namelists_only | hist_only | bless_tput | bless_mem | bless_perf)
+    if bless_perf:
+        bless_mem = True
+        bless_tput = True
+
+    bless_all_non_perf = not (namelists_only | hist_only | bless_tput | bless_mem)
+    perf_bless = bless_mem or bless_tput
+
+    expect(
```
Comment (jasonb5) [inline CIME/bless_test_results.py:226]: Let's combine these two expects.

File: CIME/bless_test_results.py [hunk 2]
```diff
@@ -216,7 +216,21 @@ def bless_test_results(
     bless_perf=False,
     **_,  # Capture all for extra
 ):
-    bless_all = not (namelists_only | hist_only | bless_tput | bless_mem | bless_perf)
+    if bless_perf:
+        bless_mem = True
+        bless_tput = True
+
+    bless_all_non_perf = not (namelists_only | hist_only | bless_tput | bless_mem)
+    perf_bless = bless_mem or bless_tput
```
Comment (jasonb5) [inline CIME/bless_test_results.py:224]: Might be worth changing `perf_bless` to `is_bless_perf`, can be confusing to have both `perf_bless` and `bless_perf`.

**Review by jasonb5 — CHANGES_REQUESTED** [review]

**Comment by jasonb5** [comment]
@jgfouca The failing E3SM system test should be fixed on master now, probably just need to re-sync.

File: CIME/bless_test_results.py (hunk 1 above)
Comment (jgfouca) [reply CIME/bless_test_results.py:226]: Done.

File: CIME/bless_test_results.py (hunk 2 above)
Comment (jgfouca) [reply CIME/bless_test_results.py]: Done.

**Review by jasonb5 — APPROVED** [review]
LGTM

**Pull request merged**